app/constants/calls.ts

```diff
@@ -6,9 +6,9 @@ import {toMilliseconds} from '@utils/datetime';
 const RefreshConfigMillis = toMilliseconds({minutes: 20});
 
 const RequiredServer = {
-    FULL_VERSION: '6.3.0',
-    MAJOR_VERSION: 6,
-    MIN_VERSION: 3,
+    FULL_VERSION: '7.1.0',
+    MAJOR_VERSION: 7,
+    MIN_VERSION: 1,
     PATCH_VERSION: 0,
```

Comment:
I think, this does not have to be changed.
cc: @cpoile

Reply:
Correct -- it won't matter, considering v2's required server version is 7.1.0. This code (and the checks) could even be removed. I'll file a ticket.